tests/integration_tests/mechanics/debug_mode/heron_input.xml

```diff
@@ -15,6 +15,7 @@
     <debug>
       <inner_samples>2</inner_samples>
       <macro_steps>2</macro_steps>
+      <dispatch_plot>True</dispatch_plot>
```

Comment:
we should add to the debug test a check for the existence of the output png (we don't need to test the image contents itself imo)

Reply:
Is there a debug test already? If not I can just add the output check when I add the test.

Reply:
I don't think there is a debug unit test but there is a debug integration test.